src/matchers/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  UUID: /^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$/,
+  DATE: /[0-3]{1}[0-9]{1}\/[0-1]{1}[0-9]{1}\/[0-9]{4}/,
```

Comment:
similarly for this, does this match a particular date structure (isostring, ymd, something like that)